cmd/buildah/unshare.go

```diff
@@ -148,6 +148,14 @@ func maybeReexecUsingUserNamespace(c *cli.Context, evenForRoot bool) {
 		}
 	}
 
+	// Copy the no_pivot_root setting, for instance when running on tmpfs.
+	if no_pivot_root, present := os.LookupEnv("BUILDAH_NOPIVOT"); !present {
```

Comment:
Validate check wants you to use noPivotRoot,  Does not like underscores for variables.

Reply:
Need to get "make validate" working here...